tests/csrc/unittests/test_sampling_layer.cu

```diff
@@ -855,6 +854,7 @@ TYPED_TEST(SamplingDecodeTest, InvalidArgsBatchTopKBatchTopPContainZero)
     delete[] top_ps;
 }
 
+#if 0
```

Comment:
Why remove it?

Reply:
The dynamic decoder doesn't support process mini-batch anymore